src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -114,14 +113,14 @@ pandocTitleBlock tit auths dat =
 
 mmdTitleBlock :: Context Text -> Doc Text
 mmdTitleBlock (Context hashmap) =
-  vcat $ map go $ sortBy (comparing fst) $ M.toList hashmap
+  vcat $ map go $ sortOn fst $ M.toList hashmap
```

Comment:
In this particular case, `sortBy` and `comparing` are better than `sortOn`, since `fst` is basically free. If you check the source of `sortOn`, you'll see that it does a bunch of mapping to save the result, but in this case, the mapping is more expensive than just recomputing the result is. (Now in src/Text/Pandoc/Shared.hs, for `rank`, this was a good change.)

Reply:
By the way, I just opened ndmitchell/hlint#860 about this.

Reply:
Interesting, thanks. "Better" is a very ambiguous measure, and I think this is an example where the improved readability might be worth it.